l10n_br_account_payment_order/views/account_payment_line.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+    Copyright (C) 2014 KMEE (http://www.kmee.com.br/)
+    @author Luis Felipe Miléo <mileo@kmee.com.br>
+    The licence is in the file __openerp__.py
+-->
+<odoo>
+
+        <record id="l10n_br_banking_account_payment_line_form" model="ir.ui.view">
```

Comment:
I think you can update this id, now this modulo's name is l10n_br_account_payment_order